packages/framework/presence/src/internalTypes.ts

```diff
@@ -79,6 +79,13 @@ export interface ValueManager<
 > {
 	// State objects should provide value - implement Required<ValueManager<...>>
 	readonly value?: TValueState;
+
+	/**
+	 * Process an update of value for remote attendee.
```

Comment:
Nit (to make explicit that we're referring to a specific entity and not a concept):
```suggestion
	 * Process an update of `value` for remote attendee.
```